### PR TITLE
Change venue selection from radio to dropdown

### PR DESCRIPTION
**Pull request by dorianjp**
# Summary | Résumé

Trello:
https://trello.com/c/dHRf3Ux8/807-update-type-of-venue-page-to-include-additional-categories-and-use-drop-down

* Changed radio selection to dropdown
* Added 3 new selection options
* css to widen form width for this page
* js updated to control select element

# Test instructions | Instructions pour tester la modification

* Register email address for poster
* Navigate to venue selection

### CODE REVIEW

**Comment by dorianjp** [comment]
[Update Type of Venue page to include additional categories and use drop-down.](https://trello.com/c/dHRf3Ux8/807-update-type-of-venue-page-to-include-additional-categories-and-use-drop-down)

File: register/templatetags/location_type_str.py [hunk 1]
```diff
@@ -17,23 +17,23 @@ def location_type_str(location_category, prefix, *args, **kwargs):
     """
     if location_category:
         if prefix == "address":
-            if location_category_type_map[location_category] == type_establishment:
+            if location_category_type_map[location_category] == type_place:
                 return _("Address of establishment")
```
Comment (dorianjp) [inline register/templatetags/location_type_str.py:21]: Text output in this is updated in the content PR #539

File: register/templates/register/location_category.html [hunk 2]
```diff
@@ -17,11 +17,11 @@
 
   {% include  "includes/flash_messages.html" %}
   <div class="step-one-info">
-    <h1>{% trans "Type of establishment or event" %}</h1>
+    <h1>{% trans "Type of place or event" %}</h1>
     <p>{% trans "Choose the type that best fits your situation." %}</p>
   </div>
   
-  {% include "register/includes/category_form.html" with submit_text=_("Next") %}
+  {% include "register/includes/category_form.html" with submit_text=_("Next") form_class="registration-category-form" %}
```
Comment (dorianjp) [inline register/templates/register/location_category.html:24]: custom form width!

File: register/static/register/js/something-else.js [hunk 3]
```diff
@@ -1,22 +1,17 @@
...
+    document.getElementById("id_category-category_description").parentElement.style.display = 'block';
+  } else {
+    document.getElementById("id_category-category_description").value = "";
+    document.getElementById("id_category-category_description").parentElement.style.display = 'none';
   }
-
-
+}
+// Add optgroup to allow text zoom in mobile html control
+const optGroup = document.createElement("optgroup");
```
Comment (dorianjp) [inline register/static/register/js/something-else.js:16]: When optgroup is added to the select element, it enables mobile to zoom the text to fit the screen when scrolling

Comment (dsamojlenko) [reply register/static/register/js/something-else.js:16]: oh cool, hadn't seen that before...

**Review by dsamojlenko — APPROVED** [review]
LGTM 💰

**Pull request merged**